### ISSUE TITLE
Additional push events for FCU buttons

### ISSUE THREAD

**Issue by frankkopp** [flybywiresim/a32nx#4718]
### Aircraft Version

Development

### Description

As we now have push events for the FCU autopilot buttons it would be great to extend these push events to all FCU buttons:
https://github.com/flybywiresim/a32nx/blob/master/docs/a320-events.md

Buttons (for each side):
FD
LS
CSTR
VOR D
WPT
ARPT
ADF1/VOR1
ADF2/VOR2

For LS, CSTR, VOR D, WPT, ARPT there is LVAR:BTN_{}_{1,2}_FILTER_ACTIVE for reading. But changing these vars only changes the button state (light on/off) but has no influence on the ND and also on the other buttons (ND buttons are exclisive).

ADF/VOR 1+2 there seems to be no event or var at all. 

### Additional info (optional)

Basically this would allow external hardware to mapp these buttons. 

Example Stream Deck: 
![image](https://user-images.githubusercontent.com/16833201/117178904-d26c9180-add2-11eb-876c-cf6b04db3e99.png)

### Discord Username (optional)

Cdr_Maverick#6475

**Comment by aguther** [comment]
You’re talking about glareshield not FCU.

**Comment by frankkopp** [comment]
> You’re talking about glareshield not FCU.

Thanks - you are right of course - it's not FCU :)
Should have looked it up - isn't it even called EFIS Control Panel?
http://www.a320dp.com/A320_DP/flight-instruments/sys-8.1.0.html

**Comment by abeauchamp** [comment]
As of today it seems that all these buttons can be mapped to events/variables. 
They are all functionnal except for the LS button. Indeed there is an issue with the BTN_LS_1_FILTER_ACTIVE LVAR. When this variable is switched to 1, it does switch the LS button on and display an ILS course pointer on the navigation display, but nothing changes on the PFD. If I ckick on the LS button with the mouse then then indicators show un on the PFD an normal.

**Comment by frankkopp** [comment]
> As of today it seems that all these buttons can be mapped to events/variables. They are all functionnal except for the LS button. Indeed there is an issue with the BTN_LS_1_FILTER_ACTIVE LVAR. When this variable is switched to 1, it does switch the LS button on and display an ILS course pointer on the navigation display, but nothing changes on the PFD. If I ckick on the LS button with the mouse then then indicators show un on the PFD an normal.

This is even documented now - the issue is indeed in the PFD (doesn't recognize the change variable): 

https://docs.flybywiresim.com/pilots-corner/a32nx-briefing/a32nx_api/#efis-control-panel

**Comment by frankkopp** [comment]
> Is there a ticket for that ?

This is the ticket :)

**Comment by abeauchamp** [comment]
Alright after looking throught the code, it seems that the BTN_LS_{ID}_FILTER_ACTIVE mentionned in the documentation (in the page you just linked) is only for the visual indicator of the button state. To interact with the button there is an H:Event called H:A320_Neo_PFD_BTN_LS_{ID}. I don't have access to my simulator right now so I'll try that later. This whole ticket might just be that the doc is lacking some info.

**Comment by frankkopp** [comment]
> Alright after looking throught the code, it seems that the BTN_LS_{ID}_FILTER_ACTIVE mentionned in the documentation (in the page you just linked) is only for the visual indicator of the button state. To interact with the button there is an H:Event called H:A320_Neo_PFD_BTN_LS_{ID}. I don't have access to my simulator right now so I'll try that later. This whole ticket might just be that the doc is lacking some info.

H: Events are not documented on purpose. Hard to use. It would be better if this also would work with LVARs as the rest.
